profileService/internal/service/service.go

```diff
@@ -0,0 +1,58 @@
+package service
+
+import (
+	"fmt"
+	models2 "github.com/kas240sx89/samples/profileService/internal/models"
+)
+
+type Service struct {
+	db DB
+}
+
+func New(database DB) *Service {
+
+	return &Service{
+		db: database,
+	}
+}
+
+func (s *Service) HealthCheck() string {
+	return "service is ok"
+}
+
+func (s *Service) GetProfile(id string) (*models2.Profile, error) {
+	profile, err := GetProfile(id)
```

Comment:
Shouldn't these be `s.db.GetProfile(id)`?  I don't see a generic `GetProfile` func.

Reply:
yes that is the way it should be, this was finished and working/ tested. not sure why the s.db. got removed or why the imports got changed to model2.